docs/source/use_CA.md

```diff
@@ -0,0 +1,205 @@
+
+## Before you begin
+
+In this tutorial, we will assume that a CA server has been configured and set up [using the CA setup instructions]. We will also show both the commands and variables used when registering an identity (a task handled by a CA admin or an identity with CA `registrar` rights), and when "enrolling" an identity (a task handled by the user of the identity).
+
+In either case, the `fabric-ca-client` must be set up, as it is used to make calls to the CA server where the identity is registered and enrolled. Because you are communicating with a node, in this case a CA, you will need to provide TLS certificates for your communication certificates. The TLS certificates will need to come from the TLS CA you spin up alongside the "organization" CA you use to generate identities, and this TLS CA is the same CA you will use to register and enroll node identities (since nodes use TLS to communicate with each other).
+
+### Specifying NodeOUs
+
+In previous versions of Fabric, identities only had two types: `client` and `peer`. The `peer` type was used for both peers and ordering nodes, while the `client` type was used both for clients (applications) and for admins, with the placement of a `client` type in a special `admincerts` folder making the identity an admin within a particular context.
+
+Now it is possible, and recommended, to encode not just `peer` or `client`, but also `orderer` or `admin` roles into the certificates generated by a CA using NodeOUs. Note that an identity can only have one of these roles, and that to enable these roles you must copy the relevant stanzas into a file called `config.yaml`. This file must exist in an `msp` folder you have created **before** you enroll an identity. Note that you can name this `msp` folder anything you want --- `msp` is the default folder name used by the Fabric CA client. If you choose another name, for example, `org1msp`, then you will have to reference this folder using the `-M` flag when enrolling the identity. More on this below.
+
+```
+ClientOUIdentifier:
+  Certificate: cacerts/localhost-7054-ca-org1.pem
+  OrganizationalUnitIdentifier: client
+PeerOUIdentifier:
+  Certificate: cacerts/localhost-7054-ca-org1.pem
+  OrganizationalUnitIdentifier: peer
+AdminOUIdentifier:
+  Certificate: cacerts/localhost-7054-ca-org1.pem
+  OrganizationalUnitIdentifier: admin
+OrdererOUIdentifier:
+  Certificate: cacerts/localhost-7054-ca-org1.pem
+  OrganizationalUnitIdentifier: orderer' > <path to your MSP>/msp/config.yaml
+```
```

Comment:
This example is if you are doing this step programmatically, but the string encapsulation is missing the opening " ' ".
The closing " ' " after the word orderer is present.  This would cause an error if copied and pasted.  Suggested change: Use full string encapsulation and note that this is the programmatic way, while also providing a way to manually generate the config.yaml file and place into the appropriate msp folder.  Programatically:
'ClientOUIdentifier:
  Certificate: cacerts/localhost-7054-ca-org1.pem
  OrganizationalUnitIdentifier: client
PeerOUIdentifier:
  Certificate: cacerts/localhost-7054-ca-org1.pem
  OrganizationalUnitIdentifier: peer
AdminOUIdentifier:
  Certificate: cacerts/localhost-7054-ca-org1.pem
  OrganizationalUnitIdentifier: admin
OrdererOUIdentifier:
  Certificate: cacerts/localhost-7054-ca-org1.pem
  OrganizationalUnitIdentifier: orderer' > <path to your MSP>/msp/config.yaml

Manual:
example config.yaml (this must be placed in the org msp folder)

ClientOUIdentifier:
  Certificate: cacerts/localhost-7054-ca-org1.pem
  OrganizationalUnitIdentifier: client
PeerOUIdentifier:
  Certificate: cacerts/localhost-7054-ca-org1.pem
  OrganizationalUnitIdentifier: peer
AdminOUIdentifier:
  Certificate: cacerts/localhost-7054-ca-org1.pem
  OrganizationalUnitIdentifier: admin
OrdererOUIdentifier:
  Certificate: cacerts/localhost-7054-ca-org1.pem
  OrganizationalUnitIdentifier: orderer